src/main/java/lottery/domain/LotteryResult.java

```diff
@@ -0,0 +1,76 @@
+package lottery.domain;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class LotteryResult {
+
+    public static final List<MatchedCountAndPrizePair> MATCHED_COUNT_AND_PRIZE_PAIRS = Arrays.asList(
+        MatchedCountAndPrizePair.of(3, 5000L),
```

Comment:
```suggestion
        MatchedCountAndPrizePair.of(3, 5_000L),
```
숫자 사이에 언더바를 추가해 가독성을 높일 수 있습니다 :)